pages/community/gsod.js

```diff
@@ -0,0 +1,144 @@
+            <br></br>
+            The other way we would consider the project successful is the number of our contributors and Docs PRs increased from 3 to 6 community members. Currently, a majority of our OSS contributor community focuses only on contributing code, but we would like to instill a greater interest in contributing to documentation that provides value for everyone.
+            </Paragraph>
+            </div>
+
+            <div className="relative mb-8 lg:mt-8">
+            <Heading level="h3" typeStyle="heading-md-semibold">
+            ⌛ Timeline
+            </Heading> 
+            <Paragraph className="mt-4 lg:pr-4">
+            The project itself will take approximately 4-6 months to complete, depending on the different levels of knowledge from diverse technical writers (TW) that might get involved. (At AsyncAPI, we want to work with any TW, regardless of their years of experience. We have a passion for mentorship, and we do not wish to have a bar that would prevent any TW from contributing to our OSS Initiative. In fact, we look forward to potentially mentoring TW(s) who are completely new to tech and making them feel welcome!)
+            <br></br>
+            For our 2 projects, we would like to request a minimum of 2 TWs, so that we can work on both the CLI/Tools and Generator Docs.
+            <br></br>
+            The timeline would look as following:
+            </Paragraph>
+            <ul>
+              <li>May: Orientation on how to contribute to AsyncAPI Inititiave, how Docs issues are organized, detail how we're migrating our CLI and Tools Docs, and assign good first-time-tickets to get each new TW contributor started.</li>
+              <li>June - August: Each TW goes through designated issues marked for both first time contributors and work set aside for GSoD 2022. Each TW starts creating documentation for their individual issues assigned/selected.</li>
+              <li>September - October: We determine if we're going to be able to complete both CLI and Tools Docs plus the Generator Docs, depending on how many TWs are in our group and how much they've been able to complete so far. We re-align priorities as needed and asses what is missing to reach our 2022 IA change goals for AsyncAPI Docs.</li>
+              <li>November: Project completion and all contributors receive some swag!</li>
+            </ul>
+            </div>
+
+            <div className="relative mb-8 lg:mt-8">
+            <Heading level="h3" typeStyle="heading-md-semibold">
+            💸 Project budget 
+            </Heading> 
+            <Paragraph className="mt-4 lg:pr-4">
+            We have set aside 2 mentors for now, for our 2 projects: improving our IA and re-structuring our Generator Docs. Should we be selected, AsyncAPI would like to request from Google a US $5000 budget for each project. For both projects, the request then totals for a $10,000 budget.
+            </Paragraph>
+
+            <table>
```

Comment:
If you want to format the table a bit more and look like some colors and border to be added. I advise using a grid instead of a table for this. Here's the [documentation](https://tailwindcss.com/docs/grid-template-columns) for it.

Reply:
Let me disagree. If this is a table, it should be a `table` element. Appearance (CSS) should not influence the type of HTML tag you choose. The same you advised her above about using `<ul>` and `<li>` tags instead of `<br>` tags applies here.

Reply:
I am also quite surprised with the behavior of the base.css applied in this element. It is adding `list-style:none` to the `<ul>` tag by default. You can check this in the developer tools in the preview URL. On running this PR locally, I found that on adding above mentioned classes, it is working fine and giving the properties it needs to show in the table.